controllers/accounts.js

```diff
@@ -109,99 +98,92 @@ router.get('/search' , function (req, res, next) {
+                ''
             ];
-        });
 
-        let sortQuery = '';
-        if (req.query.sort) {
-            sortQuery = '&sort=' + req.query.sort;
-        }
-
-        const pagination = {
-            currentPage,
-            numPages: Math.ceil(data.total / itemsPerPage),
-            baseUrl: '/users/search/?q=' + res.req.query.q + '&p={{page}}' + sortQuery
-        };
-
-        res.render('accounts/accounts', {
-            title: 'Account',
-            head,
-            body,
-            pagination,
-            user: res.locals.currentUser ||"",
-            themeTitle: process.env.SC_NAV_TITLE || 'Schul-Cloud'
+            const body = data.data.map(item => {
+                return [
+                    item.id ||"",
+                    item.username ||"",
+                    item.activated ? '✔️' : '❌',
+                    item.userId ||"",
+                    getTableActions(item, '/accounts/')
+                ];
+            });
+
+            const pagination = {
+                currentPage,
+                numPages: Math.ceil(data.total / itemsPerPage),
+                baseUrl: '/accounts/search/?q=' + res.req.query.q + '&p={{page}}'
+            };
+
+            res.render('accounts/accounts', {
+                title: 'Account',
+                head,
+                body,
+                pagination,
+                user: res.locals.currentUser ||"",
+                themeTitle: process.env.SC_NAV_TITLE || 'Schul-Cloud'
+            });
         });
-    });
 });
 
-router.patch('/:id', getUpdateHandler('accounts'));
-router.get('/:id', getDetailHandler('accounts'));
-router.delete('/:id', getDeleteHandler('accounts'));
-//router.post('/', getCreateHandler('accounts'));
+router.patch('/:id', getUpdateHandler('account'));
+router.get('/:id', getDetailHandler('account'));
+router.delete('/:id', getDeleteHandler('account'));
 
 router.get('/account/:id' , function (req, res, next) {
     const itemsPerPage = 100;
     const currentPage = parseInt(req.query.p) || 1;
 
-    api(req).get('/accounts/', {
+    api(req, { useCallback: false, json: true, version: 'v3' })
+        .get('/account', {
             qs: {
-                userId: req.params.id,
-                $limit: itemsPerPage,
-                $skip: itemsPerPage * (currentPage - 1),
-                $sort: req.query.sort
+                type: 'userId',
+                value: req.params.id,
+                limit: itemsPerPage,
+                skip: itemsPerPage * (currentPage - 1),
             }
-        }
-    ).then(data => {
-        const head = [
-            'ID',
-            'Username',
-            'Aktiviert',
-            ''
-        ];
-
-        const body = data.map(item => {
-            return [
-                item._id ||"",
-                item.username ||"",
-                item.activated ? '✔️' : '❌',
-                getTableActions(item, '/accounts/')
+        })
+        .then(data => {
```

Comment:
Another place we could use await instead.

Reply:
I only changed the spacing and without automated tests I'm not gonna change to much code.